org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/rule/RuleDescriptionPanel.java

```diff
@@ -179,13 +179,14 @@ private static void createSourceViewer(String html, Composite parent, String lan
       }
     }
 
-    var snippetElement = new SourceViewer(parent, null, SWT.BORDER);
+    var snippetElement = new SourceViewer(parent, null, SWT.BORDER | SWT.H_SCROLL);
     var gridData = new GridData();
     gridData.horizontalAlignment = SWT.FILL;
     gridData.grabExcessHorizontalSpace = true;
+    gridData.horizontalIndent = 10;
```

Comment:
Is the indentation intended? Or was it added just for testing?

Reply:
This is intended, I find it looks better like this. WDYT?